keps/sig-node/20191211-node-maintenance-lease.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Node Maintenance Lease
+authors:
+  - "@michaelgugino"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+  - sig-node
```

Comment:
iterating comments from before, IMO this should be in reverse.
KEP folder / code owner (NodeLifecycleController) -> KEP owner
other SIGs -> participants